generator/src/main/java/com/scottlogic/deg/generator/generation/SystemOutDataGeneratorMonitor.java

```diff
@@ -7,54 +7,23 @@
 import com.scottlogic.deg.generator.walker.reductive.ReductiveState;
 import com.scottlogic.deg.generator.walker.reductive.FixedField;
 
-import java.math.BigInteger;
-import java.time.Duration;
-import java.time.Instant;
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
 import java.util.stream.Collectors;
 
 public class SystemOutDataGeneratorMonitor implements ReductiveDataGeneratorMonitor {
```

Comment:
There are now many methods in here without implementation, are they still necessary or can the interface be split up so this class only has used methods?